en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -365,7 +393,19 @@ This is an additional improvement which enables hosting `accountrecoveryendpoin
     …...
     ```
 
-8.  Add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml` file.
+8.  Navigate to `<TOMCAT_HOME>/authenticationendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties` file and uncomment the following line.
+    ```
+    identity.server.service.contextURL=https://localhost:9443
+    ```
+9.  Navigate to `<TOMCAT_HOME>/authenticationendpoint/WEB-INF/classes/EndpointConfig.properties` file and change the following line,
+    ```
+    identity.server.origin=\${carbon.protocol}://\${carbon.host}:\${carbon.management.port}
+    ```
+    as follows and update the Keystore file paths on the same file accordingly.
+    ```
+    identity.server.origin=https://localhost:9443
+    ```
```

Comment:
The following is a new step yeah?

```
10. Update the keystore file paths on the same file accordingly.
```

Reply:
fixed in [8df37a9](https://github.com/wso2/docs-is/pull/2853/commits/8df37a9d2e6fee032cd30c4fad51df1d884d0bc3)